src/kp2bw/bw_serve.py

```
@@ -374,25 +375,41 @@ def has_folder(self, name: str) -> bool:
+            params["organizationId"] = organization_id
+        data = self._request("GET", "/list/object/items", params=params or None)
+        items: list[BwItemResponse] = data.get("data", [])
         return items
 
-    def create_item(self, item: dict[str, Any]) -> str:
+    def create_item(self, item: BwItemCreate) -> str:
         """Create a single vault item via HTTP and return its ID."""
-        data = self._request("POST", "/object/item", json_body=item)
+        data = self._request("POST", "/object/item", json_body=item)  # type: ignore[arg-type]
         item_id: str = data["id"]
         logger.log(VERBOSE, f"Created item {item.get('name', '?')!r} → {item_id}")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)